### PR TITLE
fix: delete on neo4j-guide

### LINKED ISSUE #3523: delete error on neo4j-guide
The neo4j-guide has an error on delete method

```
2019-08-14 17:28:31,320 ERROR [org.jbo.res.res.i18n] (Neo4jDriverIO-8-3) RESTEASY002020: Unhandled asynchronous exception, sending back 500: org.neo4j.driver.exceptions.ClientException: Invalid input 'H': expected 'i/I' (line 1, column 19 (offset: 18))
"DELETE (f:Fruit) WHERE id(f) = $id"

```

To fix the problem the sentence must be replaced to:

`                "MATCH (f:Fruit) WHERE id(f) = $id DELETE f
`

or including the **DETACH** to remove all the relations as well

`                "MATCH (f:Fruit) WHERE id(f) = $id DETACH DELETE f
`

### PR DESCRIPTION
**Pull request by barata0**
fixes #3523

### CODE REVIEW

**Comment by gsmet** [comment]
@michael-simons can you have a look? Thanks!

**Comment by michael-simons** [comment]
Hi, totally my bad, @barata0 is correct.

File: docs/src/main/asciidoc/neo4j-guide.adoc [hunk 1]
```diff
@@ -362,7 +362,7 @@ public CompletionStage<Response> delete(@PathParam("id") Long id) {
     AsyncSession session = driver.asyncSession();
     return session
         .writeTransactionAsync(tx ->
-            tx.runAsync("DELETE (f:Fruit) WHERE id(f) = $id", Values.parameters("id", id))
+            tx.runAsync("MATCH (a:Fruit) WHERE id(a) = $id DETACH DELETE a", Values.parameters("id", id))
```
Comment (michael-simons) [inline docs/src/main/asciidoc/neo4j-guide.adoc]:
```suggestion
            tx.runAsync("MATCH (f:Fruit) WHERE id(f) = $id DELETE f", Values.parameters("id", id))
```

Comment (michael-simons) [reply docs/src/main/asciidoc/neo4j-guide.adoc]: I'd prefer that we use the same alias as in the other statements.

Your completely right in regards of the `DETACH` modifier, but as we didn't create relations, I'd rather keep it simple. If I remember correctly, I had the statement include the detach, but probably broke it when removing it. Embarrassing.

**Review by michael-simons — CHANGES_REQUESTED** [review]
Please see inline comment. Thanks.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**